versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md

```diff
@@ -29,23 +29,44 @@ The following field types are available:
 
 You can define each field, supplying a unique `name` and `type`. There are also optional parameters you can specify, such as default value and non-nullable. (Some options are only relevant for certain types.)
 
-For example, here we define two `STRING` fields. (The second is nullable; the first is not nullable):
+For example, here we define two fields. In the first, we simply supply the field type (STRING). In the second, we supply the type as ENUM, specify the two possible values for the field, and then a default value:
 
 ```kotlin
 fields {
-    field(name = "ORDER_ID", type = STRING)
-    field(name = "DESCRIPTION", type = STRING, nullable = true)
+    field("PROJECT_ID", type = STRING)
+    field("PROJECT_OPERATIONAL_STATUS", type = ENUM("OPERATIONAL", "NEW_BUILD", default = "NEW_BUILD"))
 }
 ```
-## Field type parameters
+## Field parameters
+You can add one or more parameters to any field that you declare.
 
 | Parameter | Default value | Sample Usage | Value Type | Description |
 |---|---|---|---|---|
 | default | null | `field(name = "AGE", type = INT, default=20)` | The same type as the field | Set a default value for the field where not supplied on a database insert. Mandatory for `ENUM` |
```

Comment:
default is mandatory for ENUM inside the ENUM declaration. For example: field(name="COUNTRY", type=ENUM("BRAZIL", "KUWAIT","BHUTAN",default="BRAZIL")